src/Facets/StandardizedCallFacet.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import { LibDiamond } from "../Libraries/LibDiamond.sol";
+
+/// @title StandardizedCallFacet Facet
```

Comment:
could remove the last "Facet" here to match with the titles of our other contracts